jsx-runtime.d.ts

```diff
@@ -0,0 +1,15 @@
+import type { Children, Fragment, PropsWithChildren } from './index.d.ts';
+
+export function safeChildrenArray(children: Children): Array<Children>;
+
+export function jsx(
```

Comment:
Re-exporting them would force double maintaince, probably this syntax is better:

```ts
export {
  Fragment,
  createElement as jsxs,
  createElement as jsx
} from './index';
```

Reply:
Unfortunately the jsx and jsxs function signature is different from the createElement one. We can't do that, hence the tests using the `tsconfig.jsxruntime.json` ts config. Those tests, in my opinion, are necessary to ensure we don't break the contract between our implementation and the expected one by typescript.